week-1/exercises/count.js

```diff
@@ -3,4 +3,6 @@ let count = 0;
 count = count + 1;
 
 // Line 1 is a variable declaration, creating the count variable with an initial value of 0
-// Describe what line 3 is doing, in particular focus on what = is doing
+// Describe what line 3 is doing, in particular focus on what = is doing - it is reassigning the first varibale declaration 
```

Comment:
Yep, this is right 👍 
What would the new value of count be after this executes?

Reply:
It'll be 1